frontend/components/pages/CourseDetail.tsx

```diff
@@ -144,18 +149,69 @@ export function CourseDetail({
     setIsFavorited((prevState) => !prevState)
   }
 
-  const handleGetQuestions = async (videoId: string) => {
-    const res = await fetch(
-      `${process.env.NEXT_PUBLIC_BACKEND_URL}/question/${videoId}`,
-      {
-        cache: 'no-cache',
-        headers: {
-          'Content-Type': 'application/json',
+  const createQuestion = async (createQuestionParams: {
+    title: string
+    content: string
+  }) => {
+    const { title, content } = createQuestionParams
+    if (title && title.length >= 15) {
```

Comment:
```suggestion
const titleMinLength = 15
```

みたいに切り出しお願いします。

- 文字数変えるってなった時にめんどくさい
- 15がマジックナンバーで何の値なのかわからないため

https://xtech.nikkei.com/it/article/COLUMN/20070820/279836/

Reply:
@issiyda 
マジックナンバー気をつけます！
教えていただきありがとうございます！